pkg/base/BUILD.bazel

```diff
@@ -58,7 +58,10 @@ go_test(
         "node_id_test.go",
         "store_spec_test.go",
     ],
-    args = ["-test.timeout=55s"],
+    args = select({
+        "//pkg:use_ci_timeouts": ["-test.timeout=55s"],
```

Comment:
Nit: this is pretty confusing. No reason to apply the `select` and introduce this dependency if the result is the same either way. Can you update the code to only insert this `select` *if necessary* (if the test is `enormous`)?

Reply:
Indeed this makes more sense. Done.